### ISSUE TITLE
Bug report by startdde 5.6.0.15-1 on Arch Linux

### ISSUE THREAD

**Issue by carlosal1015** [linuxdeepin/developer-center#1994]
## Issue: Bug or suggest report by startdde 5.6.0.15-1 on Arch Linux

---

Good day deepin group, my intention in this issue is communicate that today I update the [package startdde](https://www.archlinux.org/packages/community/x86_64/startdde) and when I restart the system and put the password the system log out after a few seconds.

My solution was downgrade this package

```console
sudo pacman -U /var/cache/pacman/pkg/startdde-5.6.0.14-1-x86_64.pkg.tar.zst
```

And the issue vanished, and immediately show this message.

![Screen Capture_select-area_20201130185053](https://user-images.githubusercontent.com/21283014/100681168-244f4e00-3341-11eb-9e5e-07f8efb0a1d1.png)

I think that is good show this incident.

**Comment by haonanq666** [comment]
I believe this is related to #1967. I just updated using pacman -Syu and desktop crashed seconds after login. Had to go into tty and revert back to achived packages (reverted to 2020/11/01). I guess the downgraded startdde is not the most compatible with libdde-disk-mount-plugin.so, which might have been updated. 

Why is deepin so buggy? There's these crashes, that annoying minimize button not working, janky docks, and frequent OpenGL is unsafe. I really hope they can tackle these issues quickly.

**Comment by carlosal1015** [comment]
Good remark @haonanq666, today I full upgraded to startdde 5.6.0.18-1 and the the issue vanished.

The only problem is keep executed kwin_x11 (I think that this warning is clean), but this other topic.

**Comment by Saduff** [comment]
Had the same problem, deepin crashed on startup after upgrading `startdde`, but the latest `5.6.0.18-1` is working thankfully.

> Why is deepin so buggy? There's these crashes, that annoying minimize button not working, janky docks, and frequent OpenGL is unsafe. I really hope they can tackle these issues quickly.

I know, right? I've had several packages in `IgnorePkg` for a while now, because none of the later versions work properly:
  - `deepin-kwin` stuck at `5.2.0.2-4` - window manager won't start with later versions, so desktop is completely broken and unusable, applications have no titlebars among other things
  - `deepin-dock` stuck at `5.3.0.21-1` - dock won't show up with later versions (process is running, but no dock)
  - `deepin-session-shell` stuck at `5.3.0.32-1` - the shutdown interface (power icon in dock or Ctrl+Alt+Del) won't open with later versions due to a dbus permission error

I can't remember the last time deepin was so broken that I had to ignore 4 package upgrades. At least `startdde` is fixed now, so that leaves 3.

**Comment by haonanq666** [comment]
Yes, startdde is fixed. 

However, why do I feel that after login, it takes way too long for dde-dock, RIME, and just about anything to load? The load time is abnormally long. Although I seldom reboot, this is quite annoying (nearly as much as the minimize button not working!).

Also, kwin_x11 does not run at boot for me (so no window effects). I have to manually open a terminal, run ```kwin_x11 &``` and then ```disown``` to get a usable system. 

> ```deepin-dock``` stuck at ```5.3.0.21-1```

For me the latest deepin-dock pacman installs is fine (works). However, there is the odd thing that upon boot, clicking the icon in the dock does not minimize the program (i.e. chromium is open, clicking the icon in the dock should minimize chromium but it doesn't). I need to ```killall dde-dock``` to restart it, then the behavior is as expected. 

I have tried to add ```killall dde-dock``` to ```/etc/profile.d``` and ```/etc/init.d``` without reliable success too (the dock still needs to be manually killed after login).

**Comment by haonanq666** [comment]
I guess the _this_ issue is fixed, so closing is appropriate (I found out startdde introduced more errors). 

@Saduff yup so my suspend and stuff didn't work and I was tinkering #1998, and I mixed and matched packages so now things are really janky. ```deepin-session-shell``` really needed to stay at ```5.3.0.32``` or else you couldn't wake from suspend properly. 

The latest ```deepin-kwin``` works for me, but for some reason I get KDE titlebars (the blue one). Downgrading to ```5.2.0.24``` and I get white deepin title bars (even though I am in dark mode). 

```downgrade``` has been my friend these days, where I constantly change different versions. Hopefully fixes are pushed soon.
